packages/trader/src/AppV2/Components/TradeParameters/Stake/index.ts

```diff
@@ -1,4 +1,5 @@
 import Stake from './stake';
+
```

Comment:
All this empty lines between imports is a result of running autofix to sort these imports because of eslint